setup.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+# Setup script for NetworkManager WPA2-Enterprise workaround
+
+#? safer bash options
```

Comment:
What does the `?` indicate in comments?

Reply:
Ah it's just my personal comment styling. I use `#?` to indicate a brief explanation, `#NOTE:` to indicate a longer and potentially important note about the following code, and a few others. It's not important, I just personally like the organization